applications/GeoMechanicsApplication/tests/lysmer_boundary_stiff_column2d_quad/README.md

```diff
@@ -0,0 +1,31 @@
+# Lysmer absorbing boundary conditions on a column made of rectangulars
+
+This test verifies the implementation of the Lysmer absorbing boundary conditions. 
```

Comment:
From this sentence, it is not completely clear to me what the difference is between the two tests. The name of the folder suggests this is a more stiff material. The only difference I can see is that in the project parameters the `"virtual_thickness"` is different (1000 vs 0.001) between the two tests. I think that would be worth mentioning somewhere in the descriptions.

Reply:
I have updated the descriptions. From my viewpoint, the first test does not really check the boundary conditions, just P-wave propagation through the 1D column. The second test checks the P-wave reflected from the boundary.

Reply:
Just one question for my understanding: Does the absorbing BC have an influence on this wave? (This is non-blocking, just for me to understand)